<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tx_hash_input_window</class>
 <widget class="QWidget" name="tx_hash_input_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find TX by hash</string>
  </property>
  <widget class="QLineEdit" name="tx_hash_input">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>10</y>
     <width>501</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Tx hash:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_tx_by_hash_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>501</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
